misk/web/@misk/components/src/components/ErrorCalloutComponent.tsx

```diff
@@ -0,0 +1,44 @@
+import { Callout } from "@blueprintjs/core"
+import * as React from "react"
+import styled from "styled-components"
+
+/**
+ * <ErrorCalloutComponent error={props.error}/>
+ */
+
+export interface IError {
+  [key: string]: any   
+}
+
+export interface IErrorCalloutProps {
+  error: IError
+}
+
+const ErrorCallout = styled(Callout)`
+  margin: 20px 0;
+`
+
+const RawError = styled.pre`
+  text-align: left;
+`
+
+const generateStatus = (props: IErrorCalloutProps) => {
+  return props.error.response.status ? `[${props.error.response.status}]` : "[Error]"
```

Comment:
Not sure how the rest of our ts code looks, but could this be something like `[${props.error.response.status || "Error"}]`?